.github/workflows/automerge.yml

```diff
@@ -0,0 +1,11 @@
+# File managed by web3-bot. DO NOT EDIT.
```

Comment:
These should be added by the automation pipeline. Please remove all CI files with this header and instead open a PR to add a row [here](https://github.com/protocol/.github/blob/master/configs/go.json#L380)

Reply:
Done